boards/arm/ip_k66f/ip_k66f.dts

```diff
@@ -93,3 +94,10 @@
 &enet {
 	status = "okay";
 };
+
+&spi1 {
+	status = "okay";
+
+	cs-gpios = <&gpiob 10 GPIO_ACTIVE_LOW>;
```

Comment:
AFAIK spi_mcux_dspi supports PCS lines, I would not use gpio for CS on this platform.

Reply:
I've checked at least two times if I could use PCS. The PCS was not working and as fair as I understand the issue was with the SPI driver for K6x devices not driving this line at all.

I can check third time if the PCS is supported by NXP's SPI driver for K6x

Reply:
The driver definitely supports PCS; we use it on the frdm_k64f board.